events_task/test/my_app_web/controllers/event_controller_test.exs

```diff
@@ -0,0 +1,224 @@
+defmodule MyApp.EventControllerTest do
+  use MyAppWeb.ConnCase
+
+  alias MyApp.Accounts
+  alias MyAppWeb.Guardian
+  alias MyApp.Content
+
+  @valid_attrs %{
+    "description" => "TestText",
+    "timestamp" => %{
+      "day" => "1",
+      "hour" => "0",
+      "minute" => "0",
+      "month" => "1",
+      "year" => "2015"
+    }
+  }
+  @invalid_attrs %{
+    "description" => "",
+    "timestamp" => %{
+      "day" => "1",
+      "hour" => "0",
+      "minute" => "0",
+      "month" => "1",
+      "year" => "2015"
+    }
+  }
+  describe "events" do
+    def create_test_user(attrs \\ %{}) do
+      {:ok, user} =
+        attrs
+        |> Enum.into(%{email: "sergei@gmail.com"})
+        |> Accounts.create_user()
+
+      user
+    end
+
+    def create_test_event(attrs \\ %{}) do
+      {:ok, event} =
+        attrs
+        |> Enum.into(@valid_attrs)
+        |> Content.create_event()
+
+      event
+    end
+
+    test "successfully open new event page" do
+      conn = get(build_conn(), event_path(conn, :new))
+      assert conn.path_info == ["events", "new"]
+    end
+
+    test "successfully event creation with valid params" do
+      conn =
+        build_conn()
+        |> Guardian.Plug.sign_in(create_test_user())
+        |> post(event_path(conn, :create, %{"event" => @valid_attrs}))
+
+      assert redirected_to(conn) =~ "/events/"
```

Comment:
Here, you checked that controller redirected you to `/events` path - but technically nothing stops controller from redirecting to `/events` even if event was not created.
It's better to also directly check for what test description says - that some event with this attrs appeared in DB.